### PR TITLE
fix: set ownerMember in EnglishAuctionSettledEvent to previous owner

### PR DESCRIPTION
**Pull request by zeeshanakram3**
set `ownerMember` in `EnglishAuctionSettledEvent` to previous owner(not the auction winner)

### CODE REVIEW

File: query-node/mappings/src/content/nft.ts [hunk 1]
```diff
@@ -865,14 +869,16 @@ export async function contentNft_EnglishAuctionSettled({ event, store }: EventCo
...
+    store,
+    videoId.toNumber(),
+    event.blockNumber
+  )
 
   if (winnerId.toString() !== winningBid.bidder.id) {
     return inconsistentState(`English auction winner haven't placed the top bid`, { videoId, winnerId })
   }
 
-  nft.ownerMember = winningBid.bidder
```
Comment (kdembler) [inline query-node/mappings/src/content/nft.ts:874]: Why was this removed? Won't this create a bug of owner not being updated?

Comment (zeeshanakram3) [reply query-node/mappings/src/content/nft.ts:874]: This expression was redundant. The new owner has already been set before this line, in `resetNftTransactionalStatusFromVideo` [function](https://github.com/Joystream/joystream/blob/6eea1608f610ddf4a5f5588eaf6d8ca3705468a3/query-node/mappings/src/content/nft.ts#L168) (which is called from `finishAuction` [function](https://github.com/Joystream/joystream/blob/6eea1608f610ddf4a5f5588eaf6d8ca3705468a3/query-node/mappings/src/content/nft.ts#L294))

Comment (kdembler) [reply query-node/mappings/src/content/nft.ts:874]: Okay, thanks for clarification

**Review by kdembler — APPROVED** [review]
